### PR TITLE
[Data Factory]Add support for data flow

### PR DESCRIPTION
**Pull request by jackmagic313**
- Added support for Data Flow:
        * Added data flow object model and CRUD APIs
        * Added data flow debug session APIs
        * Added support for execute data flow activity type

Swagger PRs:
https://github.com/Azure/azure-rest-api-specs/pull/7069
https://github.com/Azure/azure-rest-api-specs/pull/7088
https://github.com/Azure/azure-rest-api-specs/pull/7242
https://github.com/Azure/azure-rest-api-specs/pull/7307

### CODE REVIEW

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Current version
 ###  Feature Additions
+- Added support for Data Flow:
```
Comment (hvermis) [inline sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md]: You can update the version here now - add  "## Version 4.2.0"

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
...
+        * Added data flow object model and CRUD APIs
```
Comment (hvermis) [inline sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md]: nit: Keep whitespaces the same, copy paste from other lines

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md (hunk 1 above)
Comment (jackmagic313) [reply sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md]: done

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md (hunk 2 above)
Comment (jackmagic313) [reply sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md]: I need the intend in order to add these as sub list

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Microsoft.Azure.Management.DataFactory.csproj [hunk 3]
```diff
@@ -5,12 +5,17 @@
...
+    <Version>4.2.0</Version>
     <AssemblyName>Microsoft.Azure.Management.DataFactory</AssemblyName>
     <PackageTags>Microsoft Azure resource management;Data Factory;ADF;</PackageTags>
     <PackageReleaseNotes>
   <![CDATA[
+* Added support for Data Flow:
+    - Added data flow object model and CRUD APIs
+    - Added data flow debug session APIs
+    - Added support for execute data flow activity type
 - Added property queryTimeout to tabular sources
```
Comment (isra-fel) [inline sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Microsoft.Azure.Management.DataFactory.csproj]: This line's probably for the last version. Should be removed.

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md [hunk 4]
```diff
@@ -2,7 +2,15 @@
...
 ## Current version
 ###  Feature Additions
+
+## Version 4.2.0
+###  Feature Additions
+* Added support for Data Flow:
+    - Added data flow object model and CRUD APIs
+    - Added data flow debug session APIs
+    - Added support for execute data flow activity type
 - Added property queryTimeout to tabular sources
```
Comment (isra-fel) [inline sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md]: This line belongs to 4.1.4 -- see #7734 
Please add a title and put it there.

**Review by isra-fel — CHANGES_REQUESTED** [review]
@jackmagic313 Please add link to swagger review PR(s) in this PR's description, and see my inline comments. Thanks!

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Microsoft.Azure.Management.DataFactory.csproj (hunk 3 above)
Comment (jackmagic313) [reply sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Microsoft.Azure.Management.DataFactory.csproj]: The change was just checked-in yesterday and we haven't published the nuget at all. Our intention is to include the previous change in the new version together (4.2.0)

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md (hunk 4 above)
Comment (jackmagic313) [reply sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md]: plz see above

**Comment by jackmagic313** [comment]
> @jackmagic313 Please add link to swagger review PR(s) in this PR's description, and see my inline comments. Thanks!

Replied, please help to take a look again, thx!

**Review by isra-fel — APPROVED** [review]

**Pull request merged**